site/en/docs/webstore/discovery/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: "layouts/doc-post.njk"
+title: Discovery on the Chrome Web Store
+date: 2022-03-21
+#updated: 2022-03-21
+description: >
+  An overview of how users find items on the Chrome Web Store, and how our editors select items to
+  feature.
+---
+
+We strive to make it easy for users to discover great items on the Chrome Web Store. A positive
+discovery experience means making it simple for our users to find the items they know and love, as
```

Comment:
I'm happy to take this suggestion as "our" isn't very useful here. That said, you raised a philosophical argument that I'd like to dig into a little. 

I don't agree with the assertion that "their users are not our users." When I use a podcast player on my phone, I'm using my phone (physical device), my phone's OS, the podcast player, and the podcast itself. I'm a user of all four of these components. Depending on the context, the fact that I'm an Windows Phone user may or may not be relevant to the fact that I'm also consuming my podcast feed through Libsyn. Chrome extensions users are also Chrome users because these extensions are being used in Chrome.

Reply:
I get what you're saying. I don't think extension creators would appreciate us referring to their users as our users whether it's correct or not. Please don't do it.